submissions/YuliiaHordiichuk/memory-game/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Memory Pair Game</title>
+</head>
+
+<body>
+    <ul class='cards'>
```

Comment:
Mixing single quotes and double quotes without reason is better to be avoided. Usually we use double quotes in html and single quotes in js